src/requests/campaign-requests.js

```diff
@@ -69,10 +69,7 @@ export async function fetchCampaign (id, url = "campaigns.info") {
   }
 }
 
-export async function fetchCampaignTechnology (
-  url = "/campaigns.technologies.info",
-  id
-) {
+export async function fetchCampaignTechnology (url = "/campaigns.technologies.info",id) {
```

Comment:
I'm approving this but pls ensure that optional params come after mandatory ones.
JavaScript doesn't complain but this is usually a code smell.